src/components/LoginModal/SignupForm/SignupForm.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import LoginLayout from '../LoginLayout/LoginLayout';
+import FormText from '../LoginLayout/FormText/FormText';
+import { api } from '../../../config';
+
+const SignupForm = ({
+  closeLogin,
+  goBack,
+  goToSignin,
+  inputState,
+  handleInput,
+}) => {
+  const history = useHistory();
+
+  // FIXME: 유효성검사 추가 예정
+  // pw: 8자 이상, 대소문자+숫자+특수문자
+
+  const isError =
+    inputState.password &&
+    inputState.passwordConfirmation &&
+    inputState.password !== inputState.passwordConfirmation;
```

Comment:
```suggestion
const {password , passwordConfirmation} = inputState
  const isError = 
  password && 
  passwordConfirmation && 
  password !== passwordConfirmation;
```
* 이렇게 구조분해 할당하면 더 직관적으로 key만 볼 수 있습니다